src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -37,6 +41,7 @@ const defaultProps = {
 };
 
 function ReportActionItemEmojiReactions(props) {
+    const hasErrors = !_.isEmpty(props.reportAction.errors);
```

Comment:
Why do we need to get errors separately here? Can we use the `hasErrors` from `ReportActionItem`?

Reply:
Yes, please. So we can avoid passing unnecessary data to the component.